### PR TITLE
Improve Get Started section

### PR DESCRIPTION
**Pull request by nilminiwso2**

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ strict: false
 nav:
   - Home: index.md
   - Get started:
-      - Install the Ballerina extension: get-started/install.md
+      - Install: get-started/install.md
```
Comment (anupama-pathirage) [inline en/mkdocs.yml:42]: Shouldn't we match the page titles here? Page title is `Install the Ballerina extension`?

File: en/docs/get-started/open-packages.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
 # Open Ballerina packages
 
-You can work on a single [Ballerina package](https://ballerina.io/learn/package-references/) at a time or use [Multi-Root workspaces](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) of VS Code to work on serverl related Ballerina Packages at the same time. 
+You can work on a single [Ballerina package](https://ballerina.io/learn/package-references/) at a time or use [multi-Root workspaces](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) of the VS Code editor to work on serveral related Ballerina packages at the same time.
```
Comment (anupama-pathirage) [inline en/docs/get-started/open-packages.md:3]: Is the link for `multi-root workspaces` correct? Shouldn't it be https://code.visualstudio.com/docs/editor/multi-root-workspaces

Comment (anupama-pathirage) [inline en/docs/get-started/open-packages.md:3]:
```suggestion
You can work on a single [Ballerina package](https://ballerina.io/learn/package-references/) at a time or use [multi-root workspaces](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) of the VS Code editor to work on serveral related Ballerina packages at the same time. 
```

File: en/docs/get-started/install.md [hunk 3]
```diff
@@ -1,21 +1,30 @@
...
+1. Install the Ballerina VS Code extension from the [marketplace](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina). 
+
+	> **Info:**
+	> Alternatively, you can do this from the **Extensions** activity bar in your VS Code editor. [Learn more](https://code.visualstudio.com/docs/editor/extension-marketplace#_install-an-extension).
+
+2. Open a Ballerina `(.bal)` file or a [Ballerina package](https://ballerina.io/learn/package-references/) to activate the extension.
 
-	>**Info:** When the extension is activated properly, you can see the `Ballerina SDK: <version>` in the status bar at the bottom left corner.
+	>**Info:**
+	>When the extension is activated properly, you will see `Ballerina SDK: <version>` in the status bar as shown below.
```
Comment (anupama-pathirage) [inline en/docs/get-started/install.md:24]:
```suggestion
	>When the extension is activated properly, you will see `Ballerina SDK: <version>` in the status bar, as shown below.
```

File: en/docs/get-started/open-packages.md (hunk 2 above)
Comment (anupama-pathirage) [inline en/docs/get-started/open-packages.md:3]:
```suggestion
You can work on a single [Ballerina package](https://ballerina.io/learn/package-references/) at a time or use [multi-Root workspaces](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) of the VS Code editor to work on several related Ballerina packages at the same time. 
```

File: en/docs/get-started/open-packages.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
+You can work on a single [Ballerina package](https://ballerina.io/learn/package-references/) at a time or use [multi-Root workspaces](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) of the VS Code editor to work on serveral related Ballerina packages at the same time. 
 
 ## Open a Ballerina package
 
-Use one of the following options to open a Ballerina package:
+Use one of the following options to open a Ballerina package on the VS Code editor:
 
 - From the command line, execute `code <path-to-package-dir>`.
-- From VS Code, click **File > Open Folder ...**.
+- From the VS Code editor, click **File > Open Folder ...**.
```
Comment (anupama-pathirage) [inline en/docs/get-started/open-packages.md:10]: Do we need to say `editor` always? Even vs code docs doesn't followed it with the word `editor`

https://code.visualstudio.com/docs/editor/editingevolved

**Comment by nilminiwso2** [comment]
Invalid. Replaced by new PR.

**Pull request closed without merging**